src/guides/install/local-installation.md

```diff
@@ -1,34 +1,24 @@
 # Local Installation
 
 
-The following instructions describe how to install `augur` (bioinformatics tooling) and `auspice` (our visualization app) on macOS or an Ubuntu-style Linux distribution.
-If you are using Windows, we have instructions for [installing a Linux subsystem](./windows-help) to get Nextstrain running.
-
-If you have any issues with installing Augur/Auspice using any of these methods, please [email us](mailto:hello@nextstrain.org) or submit a GitHub issue to [Augur](https://github.com/nextstrain/augur/issues) or [Auspice](https://github.com/nextstrain/auspice/issues).
-
-> Before digging in, it's worth reading [the difference between a local and container installation](./local-vs-container-install), both of which will install the components behind Nextstrain and allow you to run and visualize analyses on your computer.
+The following instructions describe how to install `augur` (bioinformatics tooling) and `auspice` (our visualization app) on macOS or an Ubuntu-style Linux distribution using Conda.
```

Comment:
It might be worth the extra emphasis in the first sentence that this doc is for installing **both** augur and auspice at once, e.g.:
```suggestion
The following instructions describe how to install `augur` (bioinformatics tooling) and `auspice` (our visualization app) **together, in the same Conda environment**, on macOS or an Ubuntu-style Linux distribution.
```